test/e2e/autoscale_test.go

```diff
@@ -320,9 +320,9 @@ func TestAutoscaleUpCountPods(t *testing.T) {
 	ctx.t.Log("The autoscaler spins up additional replicas when traffic increases.")
 	// note: without the warm-up / gradual increase of load the test is retrieving a 503 (overload) from the envoy
 
-	// Increase workload for 2 replicas for 30s
-	// Assert the number of expected replicas is between n-1 and n+1, where n is the # of desired replicas for 30s.
-	// Assert the number of expected replicas is n and n+1 at the end of 30s, where n is the # of desired replicas.
+	// Increase workload for 2 replicas for 35s
+	// Assert the number of expected replicas is between n-1 and n+1, where n is the # of desired replicas for 35s.
+	// Assert the number of expected replicas is n and n+1 at the end of 35s, where n is the # of desired replicas.
```

Comment:
I think this diff is not necessary anymore?

Reply:
Oh, it's 60s now after your change.